controller/controller.go

```diff
@@ -121,6 +121,8 @@ func respondWithError(w http.ResponseWriter, err error) {
 }
 
 func appHandler(c handlerConfig) http.Handler {
+	loadSchemas()
```

Comment:
Ideally this will end up in `pkg/schema`, but I don't think it makes sense to separate it form the controller in it's current state.